src/mds/MDSMap.cc

```diff
@@ -134,6 +134,8 @@ void MDSMap::dump(Formatter *f) const
 {
   f->dump_int("epoch", epoch);
   f->dump_unsigned("flags", flags);
+  f->dump_unsigned("ever_allowed_features", ever_allowed_features);
```

Comment:
These aren't really intended for anybody except us to look at and I don't want to devote strings and screen space to reminding users about features we don't want them turning on.